### PR TITLE
*: Add CRD creation for TFJob

### LINKED ISSUE #164: Add TFJob CRD creation to ksonnet components
tensorflow/k8s#345 separated out creation of the custom resource definition from the controller.
So we will need to update the ksonnet component for the TFJob controller to create the CRD.

### PR DESCRIPTION
**Pull request by gaocegege**
Close #164 , ref https://github.com/tensorflow/k8s/issues/281

Blocked by #169

### CODE REVIEW

File: components/tf-controller/deploy_crd.yaml [hunk 1]
```diff
@@ -111,6 +111,19 @@ roleRef:
   name: tf-job-operator
 
 
+---
```
Comment (jlewi) [inline components/tf-controller/deploy_crd.yaml]: Lets just delete deploy_crd.yaml rather than trying to keep it up to date.

**Comment by gaocegege** [comment]
@jlewi Thanks for your review, I have rebased and PTAL again.

**Comment by jlewi** [comment]
Can you explain the ":*" in the PR title?

**Comment by gaocegege** [comment]
It's a [commit convension](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#how-to-write-good-commit-messages) in [coala community](https://github.com/coala), which means the PR changes lots of code in different components. And the second commit `deploy_crd: Remove` means that commit only changes only one component: deploy_crd.yml

**Pull request merged**